services/skill_bindings.py

```
@@ -0,0 +1,1021 @@
+        """Recommend skills based on detected systems and provisioned specialists."""
+        scored: dict[str, tuple[RuntimeSkill, int, list[str]]] = {}
+
+        # Score skills by family match
+        for family in specialist_families:
+            for skill_id in self._family_map.get(family, []):
+                skill = self._skills.get(skill_id)
+                if not skill:
+                    continue
+                if skill_id in scored:
+                    s, pts, reasons = scored[skill_id]
+                    scored[skill_id] = (s, pts + 3, reasons + [f"family:{family}"])
```

Ruff (0.15.15)

[warning] Consider `[*reasons, f"family:{family}"]` instead of concatenation

Replace with `[*reasons, f"family:{family}"]`

(RUF005)